js/common.js

```diff
@@ -48,6 +48,14 @@ $(document).ready( function() {
         ToggleDiv( t_div );
     });
 
+    var options = {
+    	valueNames: [ 'project-link' ]
+    };
+    var list = new List('projects-list', options);
+    if(list.items.length <= 10) {
```

Comment:
Couple of reasons:
1- It is a frontend convenience, backend logic should not worry about it (if possible)
2- Easier to calculate the number of child elements vs. recursively counting projects and subprojects